www/css/coolhole.css

```diff
@@ -398,17 +398,107 @@
 }
 
 /* CoolPoints User Styling */
-#coinWrapper {
+#cpModal {
+  padding: 0;
+}
+#cpModalContent {
+  padding-top: 5px;
+}
+#cpModalBody {
+  padding: 0;
+}
```

Comment:
nit: we can just use bootstrap's padding classes, no? isn't like `p-0` a thing?

Reply:
also not a huge fan of the naming since we have multiple `cpModal`s. I'd also prefer classes that describe the style to avoid code repetition unless the style is very specific the element you're providing it against

Reply:
> nit: we can just use bootstrap's padding classes, no? isn't like `p-0` a thing?

Unfortunately, Bootstrap 3 doesn't have those, that's Bootstrap 4 and up

> also not a huge fan of the naming since we have multiple `cpModal`s. I'd also prefer classes that describe the style to avoid code repetition unless the style is very specific the element you're providing it against

I agree with this, mainly wanted to pin down the problems with the CSS and get it laid out properly before reorganizing everything.